lib/phantomjs/core.js

```diff
@@ -328,12 +280,12 @@ function getCriticalPathCss (options) {
   debuglog('stripped out non matching media queries')
 
   page.open(options.url, function (status) {
-    if (status !== 'success') {
```

Comment:
I prefer it like this ~ returning early; taking care of the edge case first before going to main code.

Actually looking at this now I would keep it like this but just return after the phantomExit below, then remove the unnecessary else clause and unindent that code.